tests/Integration/Behaviour/Features/Context/Domain/AliasFeatureContext.php

```diff
@@ -100,6 +104,43 @@ public function assertLastErrorIsInvalidAliasConstraint(): void
         $this->assertLastErrorIs(InvalidArgumentException::class);
     }
 
+    /**
+     * @When I delete alias :reference
+     *
+     * @param string $reference
+     */
+    public function deleteAlias(string $reference): void
+    {
+        /** @var string[] $aliasId */
+        $aliasId = $this->getSharedStorage()->get($reference);
+
+        try {
+            $this->getCommandBus()->handle(new DeleteAliasCommand(new AliasId((int) $aliasId)));
+        } catch (AliasException $e) {
+            $this->setLastException($e);
+        }
+    }
+
+    /**
+     * @When I bulk delete following aliases:
+     *
+     * @param TableNode $aliasesList
+     */
+    public function bulkDeleteAlias(TableNode $aliasesList): void
+    {
+        $aliasIds = [];
+
+        foreach ($aliasesList->getColumnsHash() as $aliasInfo) {
+            $aliasIds[] = (int) $this->getSharedStorage()->get($aliasInfo['reference']);
+        }
+
+        try {
+            $this->getCommandBus()->handle(new BulkDeleteAliasCommand($aliasIds));
+        } catch (AliasException $e) {
+            $this->setLastException($e);
+        }
+    }
```

Comment:
It is not bad idea to use table (I guess its even prefered in most cases, but not sure its worth to use it for one column only), so you could use inline step definition as we do for products, features, cart rules and probably some others which will allow reusing $referencesToIds method. The list of deletable alias for tests will never be long I guess, so in the end it will be less code and probably even more readable
```suggestion
    public function bulkDeleteAlias(string $aliasReferences): void
    {
        try {
            $this->getCommandBus()->handle(new BulkDeleteAliasCommand($this->referencesToIds($aliasReferences));
        } catch (AliasException $e) {
            $this->setLastException($e);
        }
    }
```
```